lib/membrane_live/tokens.ex

```diff
@@ -9,27 +9,45 @@ defmodule MembraneLive.Tokens do
   Refresh Token:
     - encode: generation and signing
     - decode: verification and validation
+
+  FYI: pem = Privacy-Enhanced Mail - format for storing cryptographic keys. In this case: google public keys
   """
   alias MembraneLive.Tokens.{AuthToken, GoogleToken, RefreshToken}
 
-  @google_pems_url "https://www.googleapis.com/oauth2/v1/certs"
-
   def google_decode(jwt) do
-    GoogleToken.verify_and_validate(jwt, get_signer(jwt))
+    with {:ok, signer} <- get_signer(jwt) do
+      GoogleToken.verify_and_validate(jwt, signer)
+    else
```

Comment:
If you don't specify the `else` part the `with` statement will return whatever that has not been previously matched 😉